pkgs/by-name/ri/ricochet-refresh/package.nix

```diff
@@ -3,24 +3,24 @@
 , fetchFromGitHub
 , qt5
 , openssl
-, protobuf3_20  # https://github.com/blueprint-freespeech/ricochet-refresh/issues/178
+, protobuf_21  # https://github.com/blueprint-freespeech/ricochet-refresh/issues/178
```

Comment:
it is recommend to change such overrides in all-packages.nix

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults